src/components/service/taskService.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { url } from "components/constant";
+
+const getTasks = async() => await axios.get(`${url}/allTasks`);
```

Comment:
why is async/await everywhere here? You already have an async/await request in the component where you call it